### ISSUE TITLE
движемый

### ISSUE THREAD

**Issue by kbogatyrev** [kbogatyrev/Test#49]
От двигать причастие должно быть движимый, а не движемый

**Comment by kbogatyrev** [comment]
По-видимому, все-таки движемый, а движимый -- это прилагательное.

**Comment by dash-po** [comment]
Нет-нет-нет, движемый - не существует, все грамматики русского языка пишут, что причастие движимый, так как оно от устаревшего глагола движити, а формы с Е в принципе не существует.

**Comment by kbogatyrev** [comment]
Да, действительно, в образцах спряжения ("Дополнительные сведения о типе 6") под таблицей сказано: "... эта форма записывается как _движимый_"). В самой словарной статье нет треугольника, так что программа применила стандартный алогритм. Я бы предпочел считать _движимый_ прилагательным, но придется, видимо, вставить исключение в саму программу (раз в словарной статье нет указаний).

**Comment by kbogatyrev** [comment]
Done, version 3.0.6.0
